src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/search"
+      method="get"
+      data-qa="big">
+      <label for="input" class="label label_form">
+        <div class="icons icon__big"></div>
+        <input
+          class="input label__input-big"
+          type="text"
+          data-qa="keypress"
+          name="name"
+          id="text_input--big"
+          placeholder="Try &quot;Los Angeles&quot;">
+      </label>
```

Comment:
1. apply code styling recommendations from the checklist
2. no needs to use for/id since you wrap an input with a label
```suggestion
      <label class="label label_form">
        <div class="icons icon__big"></div>
        <input
          class="input label__input-big"
          type="text"
          data-qa="keypress"
          name="name"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
```